src/node_crypto.cc

```diff
@@ -2435,7 +2545,6 @@ void SSLWrap<Base>::DestroySSL() {
     return;
 
   SSL_free(ssl_);
-  env_->isolate()->AdjustAmountOfExternalAllocatedMemory(-kExternalSize);
```

Comment:
There is no way to tell what the external size of it is with OpenSSL 1.1.0.  And it was wrong with 1.0.X in the first place.